src/parsers/row_parser.rs

```diff
@@ -50,7 +57,7 @@ pub fn parse(line: &str, state: &mut State, strategies: &Strategies) -> String {
                 table_name,
                 types: Vec::new(),
             });
-            line.to_string()
```

Comment:
This is an example of a place in the code where we were unnecessarily reallocating a string for a line that is being passed unchanged into the output.